packages/wonder-stuff-sentry/src/capture-error.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+import {collateSentryData} from "./collate-sentry-data.js";
+import {getSentry} from "./init.js";
+
+import type {SentryData} from "./types.js";
+
+/**
+ * Capture a given error in Sentry.
+ *
+ * @param {Error} error The error to be captured.
+ */
+export const captureError = (error: Error): void => {
+    const sentryData: SentryData = collateSentryData(error);
```

Comment:
@lillialexis Can you link to the code for the bits you're talking about? I've had the webapp and mobile repos open as I've put this together to ensure I am including what they need, so I'm hoping I didn't miss something significant.

Thanks!

Reply:
For ref, I was looking at this: https://github.com/Khan/mobile/blob/5e2d4422c19ecc169daa97ceecb267d7ac221b6f/react-native/shared/errors/logException.js

Reply:
The only bit that seems obstensibly different comes before the error reporting:

```
 // If it's a timeout error, or has one anywhere in its stack, don't send it to Sentry
    if (hasTimeoutError(ex)) {
        console.log(ex, simpleStringify()); // eslint-disable-line no-console
        return;
    }

    // NOTE (Lilli): If we log to the console before sending to Sentry, the console
    // log appears in the breadcrumbs
    console.error(ex, simpleStringify(maybeSentry)); // eslint-disable-line no-console
```

And could be provided by a wrapper in the mobile code.

Reply:
Right, there are those two things, which can be wrapped, and then there was the bit where I was adding the `groupByTag`/`concatenatedMessageTag`, which I see that you moved to `collateSentryData`. 

There is one more bit that is still missing, and that is where I take the list of nested errors and add them each as a context, stringifying them and their un-collated Sentry Data (in case there is something that gets overwritten by a child error). To avoid name collisions for the Contexts, I appending an iterator to the context name:
```

        /**
         * If there are wrapped source errors on the error, then we add a context
         * for each of them.
         */
        const sourceErrors = getNestedErrors(ex);
        if (sourceErrors?.length) {
            for (let i = 0; i < sourceErrors.length; i++) {
                scope.setContext(
                    `${ReservedContextNames.sourceError}${
                        i > 0 ? ` - ${i}` : '' // e.g., "Source Error - 1"
                    }`,
                    {
                        error: sourceErrors[i].toString(),
                        sentryData: simpleStringify(
                            getSentryData(sourceErrors[i]),
                        ),
                    },
                );
            }
        }
```

<img width="1102" alt="Screen Shot 2021-11-02 at 10 24 36 AM" src="https://user-images.githubusercontent.com/185017/139915138-a5dc5554-2ef7-43c1-9de3-c9aeaef06b5e.png">